<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GT Configurator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="ComPort">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>COM Port:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="WorkArea">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>871</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="RA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>RA</string>
     </property>
     <widget class="QSpinBox" name="MotorSteps_0">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16777215</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Microsteps_0">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>127</number>
      </property>
      <property name="value">
       <number>63</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Worm_0">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16777215</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Crown_0">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16777215</number>
      </property>
      <property name="value">
       <number>360</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Motor Steps</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Microsteps</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Worm teeth</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Crown teeth</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Coil_0">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AABB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ABAB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ABBA</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Coil config.</string>
      </property>
     </widget>
     <widget class="QComboBox" name="GPIO_0">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>NONE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ST4</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ST4 config.</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Motor_0">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16777215</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Motor theeth</string>
      </property>
     </widget>
     <widget class="QSlider" name="MaxSpeed_0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>50</number>
      </property>
     </widget>
     <widget class="QSlider" name="Acceleration_0">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>63</number>
      </property>
      <property name="value">
       <number>32</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Maximum speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Acceleration</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Invert_0">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Invert direction</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mount type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="MountType">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>400</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>EQ6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HEQ5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EQ5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EQ3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EQ8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AZEQ6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AZEQ5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>114GT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DOB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="isAZEQ">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>400</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>AZEQ?</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="WriteArea">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>851</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="Write">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progress">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>731</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="RA_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>261</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>DEC</string>
     </property>
     <widget class="QSpinBox" name="MotorSteps_1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16777215</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Microsteps_1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>127</number>
      </property>
      <property name="value">
       <number>63</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Worm_1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16777215</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Crown_1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16777215</number>
      </property>
      <property name="value">
       <number>360</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Motor Steps</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Microsteps</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Worm teeth</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Crown teeth</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Coil_1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AABB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ABAB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ABBA</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Coil config.</string>
      </property>
     </widget>
     <widget class="QComboBox" name="GPIO_1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>NONE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ST4</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ST4 config.</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Motor_1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16777215</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Motor theeth</string>
      </property>
     </widget>
     <widget class="QSlider" name="MaxSpeed_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>50</number>
      </property>
     </widget>
     <widget class="QSlider" name="Acceleration_1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>63</number>
      </property>
      <property name="value">
       <number>32</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Maximum speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Acceleration</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Invert_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Invert direction</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Control">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>311</width>
       <height>441</height>
      </rect>
     </property>
     <property name="title">
      <string>Control</string>
     </property>
     <widget class="QPushButton" name="Dec_P">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>DEC+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ra_P">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>RA+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ra_N">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>RA-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Stop">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Dec_N">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>230</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>DEC-</string>
      </property>
     </widget>
     <widget class="QSlider" name="Ra_Speed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ra Speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dec Speed</string>
      </property>
     </widget>
     <widget class="QSlider" name="Dec_Speed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="Tracking">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Tracking Test</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
